lib/order.rb

```diff
@@ -0,0 +1,81 @@
+require_relative "customer"
+require 'csv'
+
+class Order
+
+  attr_reader :id
+
+  attr_accessor :products, :customer, :fulfillment_status
+
+  def initialize(id, products, customer, fulfillment_status = :pending)
+    @id = id
+    @products = products #hash with "item" => price
+    @customer = customer #instannce of customer class
+    @fulfillment_status = fulfillment_status
+
+    check_status(fulfillment_status)
+  end
+
+  # #not sure if this is how i should do this,,I'm thinking not => does not work
+  def check_status(fulfillment_status)
+    valid_status = [:pending, :paid, :processing, :shipped, :complete]
+    if valid_status.include?(fulfillment_status) == false
+      raise ArgumentError.new("not a valid shipping status")
```

Comment:
it's helpful for debugging to include the invalid argument in your messages:

```suggestion
      raise ArgumentError.new("not a valid shipping status: #{fulfillment_status")
```